visidata/loaders/mysql.py

```diff
@@ -0,0 +1,114 @@
+from visidata import *
+
+__all__ = ['openurl_mysql', 'MyTable', 'MyTablesSheet']
+
+def codeToType(type_code, colname):
+    import MySQLdb as mysql
```

Comment:
Please add this to `requirements.txt`.

Reply:
OK

Reply:
Fixed in 909ab520c0b5188cd912f89a4e42d4f719a9bb37.